Sources/Winget-AutoUpdate/functions/Update-App.ps1

```diff
@@ -17,7 +17,7 @@ Function Update-App ($app) {
     Start-NotifTask -Title $Title -Message $Message -MessageType $MessageType -Balise $Balise -Button1Action $ReleaseNoteURL -Button1Text $Button1Text
 
     #Check if mods exist for preinstall/override/upgrade/install/installed/notinstalled
-    $ModsPreInstall, $ModsOverride, $ModsUpgrade, $ModsInstall, $ModsInstalled, $ModsNotInstalled = Test-Mods $($app.Id)
+    $ModsPreInstall, $ModsOverride, $ModsCustom $ModsUpgrade, $ModsInstall, $ModsInstalled, $ModsNotInstalled = Test-Mods $($app.Id)
```

Comment:
Hmm, I just noticed that a comma might be missing here...

Reply:
Yes it is.

Reply:
I'll add it on the fly.

Reply:
https://github.com/Romanitho/Winget-AutoUpdate/commit/538255f4ff7d1516b102f8a9fa5a63fcaf47cca9

Reply:
I'd like to release a stable version before adding the custom Winget source feature.